chromium/background.js

```diff
@@ -703,3 +697,11 @@ async function import_settings(settings) {
     });
   }
 }
+
+Object.assign(exports, {
+  enableMixedRulesets,
+  all_rules,
+  initializeStoredGlobals,
+});
+
+})(typeof exports == 'undefined' ? window.background = {} : exports);
```

Comment:
@cowlicks It should be `module.exports`, not simply `exports`.
```js
typeof module == 'undefined' ? window.background = {} : module.exports
```

Reply:
I'm pretty sure what I have here works fine. `exports` refers to `module.exports` see [here](https://nodejs.org/docs/latest/api/modules.html#modules_exports).

Reply:
Are you sure?

Reply:
"I'm pretty sure", would you like to check by running `rewriter.js`?